iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/SendMessagesTests.java

```diff
@@ -118,32 +118,6 @@ public void sendMessagesWithUnusualApplicationProperties() throws Exception
         this.testInstance.identity.getClient().closeNow();
     }
 
-    @Test
-    @ContinuousIntegrationTest
-    public void tokenExpiredAfterOpenButBeforeSendHttp() throws Exception
```

Comment:
This test was tied to a once deprecated and now removed connection status callback. We still have a newer connection status change callback in our device client, but it is tested elsewhere